solutions/index.md

```diff
@@ -1,46 +1,61 @@
-
-
-<div class='one spacing'></div>
-
-## Underwater navigation
-
-Our underwater modems can not only be used for communication, but also for underwater navigation. They can be used for acoustic ranging, and to form a GPS-like infrastructure underwater, aiding underwater vehicles in self-localization as well as communication. The nodes can be used as beacons for position estimation and can transparently relay information. This allows vehicles to navigate accurately as well as communicate over long distances to other vehicles, underwater installations and/or surface nodes. These nodes may be easily dropped into water in an area of operation. Once in water, the nodes self-organize into an Underwater Positioning Network (UPN). This network can then be accessed by AUVs and Remotely Operated Vehicles (ROVs) to estimate their location, navigate and communicate effectively.
-
-<div class='one spacing' id="spacing"></div>
-
-## In-water monitoring & sensing
-
-To safeguard and manage freshwater resources, effective spatial and temporal water sampling techniques are needed. Typical manual sampling used by environmental and water utility agencies is very labor-intensive and usually too sparse to capture contamination events in a timely manner. Fixed sensing buoys are able to provide real-time data but incapable of providing spatial coverage. Traditionally available AUVs are able to provide spatial coverage, but are economically and logistically expensive to operate.
-
-The [Subnero Water Assessment Network]({{ site.baseurl }}{% link products/swan.md %}) (SWAN) contains scalable, minimally propelled autonomous platforms that can carry in-situ water quality probes and operate with minimal manual support. These mobile nodes are equipped with navigation, sensing and real-time data transmission capabilities. They can autonomously execute missions provided by users and perform vertical profiling of the water column at points of interest. Users are then able to access the data and refine missions in real-time.
-
-We provide professional deployment services and operational support for its clients.
-
-<div class='one spacing'></div>
-## Customized solutions
-
-While our products are flexible enough to meet most client requirements, there are times when a client requires technology customization in order to meet some special needs. We are happy to advise clients on how best our products can be used to meet these needs, and customize the products when necessary for specific applications.
-
-<a href="https://subnero.com/contact/">Contact us</a> to find out more about customized solutions.
\ No newline at end of file
+<div class='full'>
+  {% assign soln_pages = site.pages | where:"categories","swis" %}
```

Comment:
How is this supposed to work?

We loop through all pages (`assign soln_pages = site.pages | where:"categories","swis" %`) and then we create a new card for it with the same title and subtitle but with a different thumbnail?

Reply:
Yes, that was the original idea. This was mainly used for modems I think, when there are multiple modems in different pages, it will populate them all. This one will go thru all pages and display all with category swis.

Reply:
If we don't need to use it here, we can remove the loop right? Or if we want to keep that functionality, can we move all the hardcoded title/subtitles to the page's front matter? That way if we add another page with the category it will automatically appear here with the right title etc.

Reply:
Removed the loop.